tests/private-active-active/proxy.tf

```diff
@@ -10,6 +10,13 @@ resource "aws_instance" "proxy" {
     aws_security_group.proxy.id,
   ]
 
+  monitoring = true
```

Comment:
I removed the monitoring and EBS optimized lines, but encryption should honestly be there as it will flag most any security system monitoring the accounts the test infrastructure is deployed to... also the test should try to match what would be deployed to production generally.

Reply:
Thank you for the consideration, but realistic production infrastructure is not our primary concern for the tests at this point. Please revert all changes under `/test`.

Reply:
Can you help me understand the specific concern about the configuration of the test module? It is used as part of the CI process for this module to verify its correctness. It should not be used for any deployments outside of that context, including manual deployments, so it is not clear to me in what circumstances it would trigger a monitoring event. Regardless, if this is a hard requirement for your organization then please do fork the module and modify it as you require.